src/algebra/pointwise.lean

```diff
@@ -227,6 +234,14 @@ lemma mul_subset_mul_left [has_mul α] (h : t₁ ⊆ t₂) : s * t₁ ⊆ s * t
 @[to_additive]
 lemma mul_subset_mul_right [has_mul α] (h : s₁ ⊆ s₂) : s₁ * t ⊆ s₂ * t := image2_subset_right h
 
+@[to_additive]
+lemma subset_mul_left [mul_one_class α] (s : set α) {t : set α} (ht : (1 : α) ∈ t) : s ⊆ s * t :=
```

Comment:
Can we have finset versions of these?

Reply:
Do you mind if this happens in a second PR? This one is already 400 lines long.

Reply:
Not at all! Go ahead